dashpay/src/main/proto/dashpay.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package org.dashj.platform.dashpay;
+
+message IdentityPublicKey {
+  int32 id = 1;
+  int32 type = 2;
+  int32 purpose = 3;
+  int32 securityLevel = 4;
+  bytes data = 5;
+  bool readOnly = 6;
+  int64 disabledAt = 7;
+  bytes signature = 8;
+}
+
+message Identity {
+  int32 protocolVersion = 1;
+  bytes id = 2;
+  repeated IdentityPublicKey publicKeys = 3;
+  int64 balance = 4;
+  int32 revision = 5;
+}
+
+message DashPay {
+  Identity identity = 1;
+  string username = 2;
+  bytes salt = 3;
+}
+
```

Comment:
This is a simple representation of what a wallet could hold as far as platform information.  

This is the protobuf for `DashPayWalletExtension` which will only be used by WalletTool for now.  In DashPay this information is stored in a Room DB Table.